plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -115,16 +135,11 @@ bool GazeboYarpControlBoardDriver::gazebo_init()
         }
         std::cout<<"INITIAL CONFIGURATION IS: "<<initial_config.toString()<<std::endl;
 
-        for (unsigned int i = 0; i < m_numberOfJoints; ++i) {
-#if GAZEBO_MAJOR_VERSION >= 4
-            m_jointPointers[i]->SetPosition(0,initial_config[i]);
-#else
-            gazebo::math::Angle a;
-            a.SetFromRadian(initial_config[i]);
-            m_jointPointers[i]->SetAngle(0,a);
-#endif
-        }
+        // Set initial reference
```

Comment:
I see what you means, thank you for this clarification.
The reason was to use a uniform way to set the initial value taking care of rotationale and linear joints. I'll find a better way to handle it.